src/modules/blocks/attendees/container.js

```diff
@@ -8,19 +8,33 @@ import { bindActionCreators, compose } from 'redux';
  * Internal dependencies
  */
 import { withSaveData, withStore } from '@moderntribe/common/hoc';
-import * as actions from '@moderntribe/tickets/data/blocks/attendees/actions';
-import * as selectors from '@moderntribe/tickets/data/blocks/attendees/selectors';
+import * as UIActions from '@moderntribe/tickets/data/blocks/attendees/actions';
```

Comment:
Are these actually `UIActions` and `UISelectors`? Since they're actions and selectors from the attendees block, wouldn't it make sense to just leave them as `actions` and `selectors`?